jsontests/src/lib.rs

```diff
@@ -176,7 +272,25 @@ impl evm_runtime::tracing::EventListener for EventListener {
 				index: index.clone(),
 				value: value.clone(),
 			}),
-			_ => (),
+			RuntimeEvent::StepResult {
+				result,
+				return_value: _,
+			} => {
+				let new_event = Event::Step {
```

Comment:
`Event::Step` will need to be updated with the additional gas and depth fields. I didn't do it in this PR so as to not break compatibility with the Move output. The gas and depth data is present in `current_step`, it just needs to be passed on to `Event::Step` when the new fields are added.